mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/search/ElasticSearchSearcher.java

```diff
@@ -144,4 +156,21 @@ private int computeRequiredSize(Optional<Long> limit) {
         }
     }
 
+    public class SearchResult {
```

Comment:
Can't we find a better name for this? Because we have two occurences of **SearchResult**...

**SearchResults** is the minimum  ;-)